fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBReverseDirectoryCache.java

```diff
@@ -371,6 +371,15 @@ public void rebuild(LocatableResolver scope) {
         }
     }
 
+    /**
+     * Wait for any asynchronous work started at object creation time to complete. This should only be
+     * used for tests in order to avoid spurious conflicts.
+     */
+    @VisibleForTesting
+    public void waitUntilReady() {
```

Comment:
Since the `@VisibleForTesting` doesn't really help when, say, your favorite IDE is autocompleting potential methods, maybe it would be a good idea to also name this method `waitUntilReadyForTesting` (unless we can restrict the use of methods tagged with this automatically?).